### PR TITLE
choose if to configure the presto after server install

### PR DESCRIPTION
**Pull request by RonenHoffer**

### CODE REVIEW

File: prestoadmin/package.py [hunk 1]
```diff
@@ -100,10 +100,7 @@ def _rpm_install(package_path):
 
 
 def _nodeps_rpm_option():
-    nodeps = ''
-    if env.nodeps:
-        nodeps = '--nodeps '
-    return nodeps
+    return '--nodeps ' if env.nodeps else ''
```
Comment (kokosing) [inline prestoadmin/package.py]: please extract this refactor as separate commit

File: prestoadmin/server.py [hunk 2]
```diff
@@ -375,16 +375,23 @@ def install(rpm_specifier):
                         should ignore checking Presto rpm package
                         dependencies. Equivalent to adding --nodeps
                         flag to rpm -i.
+
+        --no-config -   pass this in order to avoid server config after installation
```
Comment (kokosing) [inline prestoadmin/server.py]: did you mean to `avoid update`? Maybe switch should be named `--no-config-update`?

File: prestoadmin/package.py (hunk 1 above)
Comment (RonenHoffer) [reply prestoadmin/package.py]: will do

File: prestoadmin/server.py (hunk 2 above)
Comment (RonenHoffer) [reply prestoadmin/server.py]: works for me - renaming switch and updating comment

File: prestoadmin/main.py [hunk 3]
```diff
@@ -316,6 +316,14 @@ def parser_for_options():
         help=SUPPRESS_HELP
     )
 
+    parser.add_option(
+        '--no-config-update',
+        action='store_true',
+        dest='no_config_update',
+        default=False,
+        help=SUPPRESS_HELP
```
Comment (kokosing) [inline prestoadmin/main.py]: Why not to provide actual help here?

Comment (RonenHoffer) [reply prestoadmin/main.py]: will do

**Comment by RonenHoffer** [comment]
added test and fix comments

File: tests/unit/test_server.py [hunk 4]
```diff
@@ -315,11 +315,26 @@ def test_specifier_as_local_path_with_file_scheme_found_locally(self):
...
+        self.assertFalse(mock_update.called)
+        mock_sudo.assert_called_with('getent passwd presto', quiet=True)
+
+    @patch('prestoadmin.server._do_config', return_value=True)
     @patch('prestoadmin.server.sudo')
     @patch('prestoadmin.server.package.deploy_install')
     @patch('prestoadmin.server.update_configs')
     def test_deploy_install_configure(self, mock_update, mock_install,
-                                      mock_sudo):
+                                      mock_sudo, mock_config):
```
Comment (kokosing) [inline tests/unit/test_server.py]: make it a single line:
```
 def test_deploy_install_configure(self, mock_update, mock_install, mock_sudo, mock_config):
```

File: tests/unit/test_server.py [hunk 5]
```diff
@@ -315,11 +315,26 @@ def test_specifier_as_local_path_with_file_scheme_found_locally(self):
     def test_specifier_as_local_path_with_file_scheme_not_located(self):
         self.check_rpm_specifier_with_location(rpm_specifier='file:///path/to/rpm', location='none')
 
+    @patch('prestoadmin.server._do_config', return_value=False)
+    @patch('prestoadmin.server.sudo')
+    @patch('prestoadmin.server.package.deploy_install')
+    @patch('prestoadmin.server.update_configs')
+    def test_deploy_install_without_configure(self, mock_update, mock_install,
```
Comment (kokosing) [inline tests/unit/test_server.py]: no need to wrap the line

File: tests/unit/test_server.py (hunk 4 above)
Comment (RonenHoffer) [reply tests/unit/test_server.py]: Done

File: tests/unit/test_server.py (hunk 5 above)
Comment (RonenHoffer) [reply tests/unit/test_server.py]: Done

**Comment by kokosing** [comment]
Merged, thanks!
